a_Calculate_Centers.R

```diff
@@ -42,127 +52,74 @@ a_Calculate_Centers_list <- list(
     pattern = map(US_states_territories)
   ),
   
-  # get distinct HUC4s as a list
+  # get distinct HUC4s as a list from the previous target
   tar_target(
     name = HUC4_list,
     command = HUC4_dataframe %>% 
       distinct() %>% 
       pull("huc4"),
```

Comment:
Is there a chance that duplicates could sneak in this way? I'm definitely not certain, just a thought. If you think it's a risk then maybe `HUC4_dataframe %>% pull("huc4") %>% unique()`

Reply:
oooo I like this. I'm going to update!